src/pycram/costmaps.py

```diff
@@ -785,7 +785,7 @@ def get_aabb_for_link(self) -> AxisAlignedBoundingBox:
             link_pose_trans = self.link.transform
             inverse_trans = link_pose_trans.invert()
             prospection_object.set_orientation(inverse_trans.to_pose())
-            return self.link.get_axis_aligned_bounding_box()
+            return self.link.get_bounding_box()
```

Comment:
Wouldn't this mean that the bounding box is not necessarily axis aligned?

Reply:
yes you are right I removed it and added two methods one for axis_ligned and one for rotated